generator-mediator-js/app/prompts.js

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+var config = require('./config')
+
+exports.promptsList = (enablePrompts) => {
+  var list = [
+    {
+      when: function() { return enablePrompts },
+      type: 'confirm',
+      name: 'enablePrompts',
+      message: 'Would you like to be prompted for config?',
```

Comment:
Do we need this question when you could rather use the `--useDefaults` option. I was confused by it.